docs/docs/basic-usage.md

```diff
@@ -67,19 +67,44 @@ It will automatically find a suitable version constraint **and install** the pac
 ## Using your virtual environment
 
 By default, poetry creates a virtual environment in `{cache-dir}/virtualenvs` (`{cache-dir}\virtualenvs` on Windows).
+You can change the [`cache-dir`](/docs/configuration/#cache-dir) value by editing the poetry config.
+Additionally, you can use the [`virtualenvs.in-project`](/docs/configuration/#virtualenvs.in-project) configuration variable
+to create virtual environment within your project directory.
+
+
 There are several ways to run commands within this virtual environment.
 
+
 ### Using `poetry run`
 
 To run your script simply use `poetry run python your_script.py`.
 Likewise if you have command line tools such as `pytest` or `black` you can run them using `poetry run pytest`.
 
 ### Activating the virtual environment
 
-The easiest way to activate the virtual environment is `poetry shell`.
+The easiest way to activate the virtual environment is to create a new shell with `poetry shell`.
+To deactivate the virtual environment and exit this new shell type `exit`.
+To deactivate the virtual environment without leaving the shell use `deactivate`.
+
+!!!note
+
+	**Why a new shell?**
+ 	Python cannot set or change environment variables in the current shell,
+	Therefore, poetry has to create a new shell with the `PYTHONHOME` and `PATH`
+	environment variables pointing to the virtual environment.
+
 
-Alternatively you can manually activate the virtual environment by running `source {path_to_venv}/bin/activate` (`source {path_to_venv}\Scripts\activate.bat` on Windows).
+Alternatively, to avoid creating a new shell, you can manually activate the
+virtual environment by running `source {path_to_venv}/bin/activate` (`source {path_to_venv}\Scripts\activate.bat` on Windows).
 To get the path to your virtual environment run `poetry env info --path`.
+You can also combine these into a nice one-liner, `source $(poetry env info --path)/bin/activate`
+To deactivate this virtual environment simply use `deactivate`.
+
+|                   | Linux/Mac                                      | Windows                                     | Exit/Deactivate |
+|-------------------|------------------------------------------------|---------------------------------------------|-----------------|
+| New Shell         | `poetry shell`                                 | `poetry shell`                              | `exit`          |
+| Manual Activation | `source {path_to_venv}/bin/activate`           | `source {path_to_venv}\Scripts\activate.bat`| `deactivate`    |
+| One-liner         | `source $(poetry env info --path)/bin/activate`|                                             | `deactivate`    |
```

Comment:
```suggestion
| One-liner         | ```source`poetry env info --path`/bin/activate```|                                             | `deactivate`    |
```
This might be more compliant with posix.